lib/ui/widgets/custom_feedback.dart

```diff
@@ -0,0 +1,133 @@
+import 'package:feedback/feedback.dart';
+import 'package:flutter/material.dart';
+
+/// A data type holding user feedback consisting of a feedback type, free from
+/// feedback text, and a sentiment rating.
+class CustomFeedback {
+  CustomFeedback({
+    this.feedbackType,
+    this.feedbackText,
+  });
+
+  FeedbackType feedbackType;
+  String feedbackText;
+
+  @override
+  String toString() {
+    return {
+      'feedback_type': feedbackType.toString(),
+      'feedback_text': feedbackText,
+    }.toString();
+  }
+
+  Map<String, dynamic> toMap() {
+    return <String, dynamic>{
+      'feedback_type': feedbackType.toString(),
+      'feedback_text': feedbackText,
+    };
+  }
+}
```

Comment:
This could go in the models folder I think 😄

Reply:
Yes totally